cmplxs/cmplxs.go

```diff
@@ -395,23 +413,26 @@ func MinAbsIdx(s []complex128) int {
 }
 
 // Mul performs element-wise multiplication between dst
-// and s and stores the result in dst. Panics if the
-// lengths of s and t are not equal.
+// and s and stores the result in dst.
+// It panics if the argument lengths do not match.
 func Mul(dst, s []complex128) {
 	if len(dst) != len(s) {
-		panic("cmplxs: slice lengths do not match")
+		panic(badLength)
```

Comment:
```suggestion
		panic(badDstLength)
```

Reply:
I agonised about these; in the case of `Mul` and other work-in-place functions, I think that `badLength` is actually a better message. It looks surprising because of the name of the source/destination parameter, but I think that it would be more confusing if you saw the message "destination slice length does not match input" for a call `floats.Mul(a, b)` when you think of both `a` and `b` as inputs.

Reply:
Ok, seems you gave it more thought that I did :-)